README.md

```diff
@@ -16,15 +16,16 @@
 
 Take a look at this [C++ project template](https://github.com/lukka/CppCMakeVcpkgTemplate) that applies all the following instructions, but also shows how to create a __pure__ workflow without using special GitHub action that you cannot run locally on your development machine, but directly using the tools (`CMake`, `Ninja`, `vcpkg`, `C++` compilers) you already use daily.
 
-# [The **run-cmake@v10** action for using CMake with CMakePreset.json on GitHub workflows](https://github.com/marketplace/actions/run-cmake)
+# [The **run-cmake@v10** action for using CMake with CMakePresets.json on GitHub workflows](https://github.com/marketplace/actions/run-cmake)
 
-The **run-cmake** action runs [CMake](https://cmake.org) on GitHub workflows by leveraging the configurations specified into [CMakePreset.json](https://cmake.org/cmake/help/latest/manual/cmake-presets.7.html).
+The **run-cmake** action runs [CMake](https://cmake.org) on GitHub workflows by leveraging the configurations specified into [CMakePresets.json](https://cmake.org/cmake/help/latest/manual/cmake-presets.7.html).
 
 Good companions are the [run-vcpkg](https://github.com/marketplace/actions/run-vcpkg) action and the [get-cmake](https://github.com/marketplace/actions/get-cmake) action.
 
 Special features which provide added value over a pure workflow are:
   - automatic dump of log files created by `CMake` (e.g., `CMakeOutput.log`) and `vcpkg`. The content of those files flow into the workflow output log. Customizable by the user.
-  - annotations for `CMake` errors/warnings and for build (`gcc`/`msvc`/`clang`) errors/warning are created inline in the changed source files the build run for, e.g.: ![Annotation](./docs/imgs/annotation.png)
+  - annotations for `CMake` errors/warnings and for build (`gcc`/`msvc`/`clang`) errors/warning are created inline in the changed source files the build run for, e.g.:  
+  ![Annotation](./docs/imgs/annotation.png)
```

Comment:
Makes sure that the embedded image is always shown on a separate line. Currently, depending on the browser window size, the image might be shown in the same line which looks rather weird:
![Image shown in same line screenshot](https://user-images.githubusercontent.com/11685886/147608065-16b9b7e5-5c23-413f-80fe-721739a10cdb.png)